### DISCUSSION TITLE
阿里云盘更新内容，alist一直不更新

### DISCUSSION THREAD

**Discussion by lidalifan** [AlistGo/alist#2928] [Q&A] [answered]
阿里云盘更新内容，alist一直看不到新的内容 只能看到老内容

**Comment by anwen-anyi** [comment, marked as answer]
阿里云盘内容更新多久了？ alist还未显示....
(可以手动刷新，在你有文件变更的文件夹里面 然后右下角三个点儿点一下能看到刷新按钮，前提是要登录管理员哦)

**Reply by gaidehaha** [reply]
点了刷新还是没有，

**Comment by NovaShen0724** [comment]
需要手动刷新，在文件夹页面或者后台存储页面都可以

**Comment by NovaShen0724** [comment]
完全不来关闭问题的。。。

**Reply by anwen-anyi** [reply]
哈哈 有没有可能 大多数人不知道 或者不知道怎么关闭

**Comment by tyhunter** [comment]
或者把过期时间设置为0，就可以做到秒更新了

**Reply by anwen-anyi** [reply]
🙅🏻🙅🏻🙅🏻🙅🏻号不想要啦?一直刷新😂😂😂

**Reply by KJFKA** [reply]
> 🙅🏻🙅🏻🙅🏻🙅🏻号不想要啦?一直刷新😂😂😂

如果设置0会被封号吗？

**Reply by tyhunter** [reply]
> 🙅🏻🙅🏻🙅🏻🙅🏻号不想要啦?一直刷新😂😂😂

一直这么用，一个多月了还没被封，转存完马上出现在infuse上爽，加上也不充钱+注册送的几百G转存TG上的分享看看够用了，阿里真要扬了那就扬了吧

**Reply by KJFKA** [reply]
web代理需要打开吗？选择302还是本地代理还是代理地址？这些是不是影响播放视频的速度？

**Reply by tyhunter** [reply]
> web代理需要打开吗？选择302还是本地代理还是代理地址？这些是不是影响播放视频的速度？

不用，默认的302即可，这样可以直接用阿里云盘的直链播放，不用本地再中转一遍

**Reply by eaglexiang** [reply]
@tyhunter  2024年来问一句，你号还在吗

**Reply by tyhunter** [reply]
> @tyhunter 2024年来问一句，你号还在吗

还在，不过我后面改infuse直接挂载了，基本不用alist，infuse没有地方让我更改刷新时间，但基本上也是存完秒出现

**Reply by pongfcnkl** [reply]
0是一直不刷新，得设置-1才是一直刷新
